ts/components/icon/lucide.ts

```diff
@@ -1,44 +1,79 @@
 export enum LUCIDE_ICONS_UNICODE {
-  EXTERNAL_LINK_ICON = '',
-  COPY = '',
-  REPLY = '',
-  REFRESH_CW = '',
-  CIRCLE_ARROW_DOWN = '',
-  TRASH2 = '',
-  LOG_OUT = '',
+  ARROW_UP = '',
```

Comment:
Sorting order is a tiny bit off, would be good to sort before merging

Reply:
made fixes, I think that's ok now?